feature1.sql

```diff
@@ -0,0 +1,60 @@
+CREATE DATABASE Primary3_CiaraS;
+
+USE Primary3_CiaraS;
+
+CREATE TABLE Team (
+	team_id MEDIUMINT PRIMARY KEY AUTO_INCREMENT,
+    team_type VARCHAR(255)
+    );
+    
+CREATE TABLE Employee (
+	employee_id MEDIUMINT PRIMARY KEY AUTO_INCREMENT,
+    employee_name VARCHAR(255),
+    salary MEDIUMINT,
+    bank_account_number CHAR(8),
+    national_insurance_number CHAR(9),
+    team_id MEDIUMINT
+	);
+    
+CREATE TABLE SalesEmployee (
+	sales_employee_id MEDIUMINT PRIMARY KEY AUTO_INCREMENT,
+    employee_id MEDIUMINT,
```

Comment:
having this as a FK makes this a 1:many relationship, is that what we want to have? think about what sort of test data you may be able to enter in the employee_id column